frontend/tmp.env

```diff
@@ -1 +1 @@
-REACT_APP_BACKEND_SERVER="https://your-remote-dev-server.com"
\ No newline at end of file
+REACT_APP_BACKEND_SERVER="https://dev-server.idurarapp.com"
```

Comment:
revert this line `REACT_APP_BACKEND_SERVER="https://your-remote-dev-server.com"`
and create new file .env and then include  `REACT_APP_BACKEND_SERVER="https://dev-server.idurarapp.com"`